main.py

```diff
@@ -736,21 +743,32 @@ def _install_flows(
             self.delete_matched_flows(
                 flow_dicts, {switch.id: switch for switch in switches}
             )
-        self._send_flow_mods(switches, flow_mods, flows, reraise_conn, send_barrier)
+        self._send_flow_mods(
+            switches,
+            flow_mods,
+            flows,
+            owners,
+            commands,
+            reraise_conn,
+            send_barrier
+        )
 
     def _send_flow_mods(
         self,
         switches,
         flow_mods,
         flows,
+        owners,
+        commands,
```

Comment:
I think passing the `commands` here inflated it a bit since a command can be derived from the flow mod (cheaply). I mean, the new code is it's aligned with the existing structured, but it's introducing a somewhat repetitive list of values. I think we should keep `build_command_from_flow_mod(flow_mod)` as we used to have. Let me know you had any other major reasons.

Reply:
My reasoning at the time was for uniformity, specifically in regards to the tuple used in _flow_mods_sent. But with your suggestion of packing that into a dataclass, I could feed `_send_flow_mods` a list of those instead.